roles/matrix-bridge-appservice-slack/defaults/main.yml

```diff
@@ -0,0 +1,92 @@
+# matrix-appservice-slack is a Matrix <-> Slack bridge
+# See: https://github.com/matrix-org/matrix-appservice-slack
+
+matrix_appservice_slack_enabled: true
+
+matrix_appservice_slack_docker_image: "cadair/matrix-appservice-slack:latest"
+
+matrix_appservice_slack_base_path: "{{ matrix_base_data_path }}/appservice-slack"
```

Comment:
In other bridges, we're separating bridge configuration and data paths.

I'd introduce a few new variables:

```yaml
matrix_appservice_slack_config_path: "{{ matrix_appservice_slack_base_path }}/config"
matrix_appservice_slack_data_path: "{{ matrix_appservice_slack_base_path }}/data"
```

.. and then update everything to use those:
- `setup_install.yml` - needs to create all the paths
- `init.yml` registration needs to refer to `matrix_appservice_slack_config_path/registration.yaml`
- etc.